admin/src/documents/pages/blocks/KeyFactsItemBlock.tsx

```diff
@@ -0,0 +1,61 @@
+import { Field, FinalFormInput } from "@comet/admin";
+import { BlocksFinalForm, createCompositeBlock, createCompositeSettings } from "@comet/blocks-admin";
+import { createRichTextBlock, SvgImageBlock } from "@comet/cms-admin";
+import { KeyFactsItemBlockData } from "@src/blocks.generated";
+import { LinkBlock } from "@src/common/blocks/LinkBlock";
+import { FormattedMessage } from "react-intl";
+
+const DescriptionRichTextBlock = createRichTextBlock({
+    link: LinkBlock,
+    rte: {
+        maxBlocks: 1,
+        supports: ["bold", "italic", "sub", "sup", "non-breaking-space", "soft-hyphen"],
+    },
+    minHeight: 0,
+});
+
+export const KeyFactsItemBlock = createCompositeBlock(
+    {
+        name: "KeyFactsItem",
+        displayName: <FormattedMessage id="keyFactsItemBlock.displayName" defaultMessage="Key Fact" />,
+        blocks: {
+            icon: {
+                block: SvgImageBlock,
+                title: <FormattedMessage id="keyFactsItemBlock.icon" defaultMessage="Icon" />,
+            },
+            settings: {
```

Comment:
```suggestion
            $settings: {
```

We normally use $ as a convention for these "virtual blocks"